_api-reference/index-apis/stats.md

```diff
@@ -644,4 +644,17 @@ GET /testindex*/_stats?expand_wildcards=open,hidden
 ```json
 GET /testindex/_stats?level=shards
 ```
-{% include copy-curl.html %}
\ No newline at end of file
+{% include copy-curl.html %}
+
+## Concurrent segment search
+
+Starting with OpenSearch 2.10, you can perform [concurrent segment search](search-plugins/concurrent-segment-search/), in which each shard-level request will search the segments in parallel during query phase. If you enable concurrent segment search, the Index Stats API response will contain several additional fields with statistics about slices (units of work executed by a thread). For more information about slices, see [Concurrent segment search]({{site.url}}{{site.baseurl}}/search-plugins/concurrent-segment-search#searching-segments-concurrently).
```

Comment:
Small nit: These stats will show up if the feature flag is enabled for concurrent search, regardless of if the cluster/index settings for concurrent search are enabled.